playbooks/openshift/deploy_aosi.yml

```diff
@@ -0,0 +1,18 @@
+# #### Dependencies
+#
+# - setup.yml
+# - repo_from_source.yml
+#
+# #### Install ose from built rpms
+# ## defined in install_ose.yml ##
+#
+# - drop yum repo config onto all nodes
+# - yum install packages
+# - configure and run atomic-openshift-installer
+
+---
+- hosts: all
+  sudo: '{{ sudo | default("no") }}'
```

Comment:
Possibly. I didn't even know about the 'become' statement, thanks! However, it does say that sudo and becomes are equivalent in 2.x (https://docs.ansible.com/ansible/become.html#for-those-from-pre-1-9-sudo-and-su-still-work). I'll make a note to update it soon.